docs/operator-manual/applicationset/Generators-Pull-Request.md

```diff
@@ -186,6 +188,37 @@ spec:
         namespace: default
 ```
 
+And, here is a robust Kustomize example;
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: myapps
+spec:
+  generators:
+  - pullRequest:
+    # ...
+  template:
+    metadata:
+      name: 'myapp-{{branch}}-{{number}}'
+    spec:
+      source:
+        repoURL: 'https://github.com/myorg/myrepo.git'
+        targetRevision: '{{head_sha}}'
+        path: kubernetes/
+        kustomize:
+          nameSuffix: {{branch}}
+          commonLabels:
+            app.kubernetes.io/instance: {{branch}}-{{number}}
+          Images:
```

Comment:
The capital I surprises me. Should this be lower?

Reply:
derp!